chat/src/main/java/ChatManagement.java

```diff
@@ -0,0 +1,22 @@
+import static com.codeborne.selenide.Selectors.byText;
+import static com.codeborne.selenide.Selenide.$;
+import static org.exoplatform.platform.qa.ui.selenium.locator.chat.ChatLocator.ELEMENT_CHAT_LIST_MSG;
+import static org.exoplatform.platform.qa.ui.selenium.locator.chat.ChatLocator.ELEMENT_CHAT_MESSAGE_INPUT;
+
+import com.codeborne.selenide.Condition;
+
+import org.exoplatform.platform.qa.ui.selenium.TestBase;
+
+public class ChatManagement {
+  private final TestBase testBase;
+
+  public ChatManagement(TestBase testBase) {
+    this.testBase = testBase;
+  }
+
+  public void sendMessageInRoomOrSpace(String room, String message) {
+    $(byText(room)).click();

```

Comment:
the xpath of a room created is variable, so it's easier to find this room byText and it will not cause problem because it isn't static or it don't change when we change langage